app/components/Views/LedgerConnect/index.tsx

```diff
@@ -0,0 +1,288 @@
+              title: strings('ledger.toast_bluetooth_connection_error_title'),
+              description: strings(
+                'ledger.toast_bluetooth_connection_error_subtitle',
+              ),
+              id: 'ledger_bluetooth_disconnect_toast_message',
+              status: 'error',
+            }),
+          );
+          navigation.navigate('SelectHardwareWallet');
+          break;
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [ledgerError]);
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <View style={styles.connectLedgerWrapper}>
+        <Image
+          source={useAssetFromTheme(
+            ledgerDeviceLightImage,
+            ledgerDeviceDarkImage,
+          )}
+          style={styles.ledgerImage}
+        />
+        <Text bold style={styles.connectLedgerText}>
+          {strings('ledger.connect_ledger')}
+        </Text>
+        <View style={styles.imageContainer}>
+          <Image
+            source={useAssetFromTheme(
+              ledgerConnectLightImage,
+              ledgerConnectDarkImage,
+            )}
+            style={styles.coverImage}
+          />
+        </View>
+        <View style={styles.textContainer}>
+          <View style={styles.lookingForDeviceContainer}>
+            <Text style={styles.lookingForDeviceText} bold>
+              {!isAppLaunchConfirmationNeeded
+                ? strings('ledger.looking_for_device')
+                : strings('ledger.open_eth_app')}
+            </Text>
+            {!selectedDevice && (
+              <ActivityIndicator style={styles.activityIndicatorStyle} />
+            )}
+          </View>
+          {!isAppLaunchConfirmationNeeded ? (
+            <>
+              <Text style={styles.instructionsText}>
+                {strings('ledger.ledger_reminder_message')}
+              </Text>
+              <Text style={styles.ledgerInstructionText}>
+                {strings('ledger.ledger_reminder_message_step_one')}
+              </Text>
+              <Text style={styles.ledgerInstructionText}>
+                {strings('ledger.ledger_reminder_message_step_two')}
+              </Text>
+              <Text style={styles.ledgerInstructionText}>
+                {strings('ledger.ledger_reminder_message_step_three')}
+              </Text>
+              <Text style={styles.howToInstallEthAppText} bold big link>
+                {strings('ledger.how_to_install_eth_app')}
+              </Text>
+            </>
+          ) : (
+            <Text>
+              {strings('ledger.open_eth_app_message_one')}{' '}
+              <Text bold>{strings('ledger.open_eth_app_message_two')} </Text>
+            </Text>
+          )}
+        </View>
+        <View style={styles.bodyContainer}>
+          {!isAppLaunchConfirmationNeeded && (
+            <Scan
+              onDeviceSelected={(ledgerDevice) =>
+                setSelectedDevice(ledgerDevice)
+              }
+            />
+          )}
```

Comment:
Nit: Prefer ternary operator

```suggestion
          {!isAppLaunchConfirmationNeeded ? (
            <Scan
              onDeviceSelected={(ledgerDevice) =>
                setSelectedDevice(ledgerDevice)
              }
            />
          ) : null}
```